modules/OFConnectionManager2/module/src/cxn_instance.c

```diff
@@ -81,6 +81,16 @@ cxn_try_to_connect(connection_t *cxn);
 /* Maximum number of messages to send per write callback */
 #define MAX_WRITE_MSGS 32
 
+/* Former cxn outstanding async count
+ * This counter count all the async pending operations of former connection(s).
+ * The bundle tasks and clear iteration tasks should be blocked if this counter
+ * is not zero.
+ * The pending operations of active connection(s) won't be added to this counter
+ * so that the bundle task can continue if the bundle support more than one
+ * outstanding operation.
```

Comment:
do we support more than one outstanding operation right now?  if we don't, let's just change the last sentence to "The pending operations of currently active connection are not added to this counter."